perception_launch/launch/object_segmentation/ground_segmentation/ground_segmentation.launch.py

```diff
@@ -0,0 +1,303 @@
+# Copyright 2020 Tier IV, Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import launch
+from launch.actions import DeclareLaunchArgument
+from launch.actions import IncludeLaunchDescription
+from launch.actions import OpaqueFunction
+from launch.actions import SetLaunchConfiguration
+from launch.conditions import IfCondition
+from launch.conditions import UnlessCondition
+from launch.launch_description_sources import PythonLaunchDescriptionSource
+from launch.substitutions import LaunchConfiguration
+from launch_ros.actions import ComposableNodeContainer
+from launch_ros.actions import LoadComposableNodes
+from launch_ros.descriptions import ComposableNode
+from launch_ros.substitutions import FindPackageShare
+import yaml
+
+
+def get_vehicle_info(context):
+    path = LaunchConfiguration("vehicle_param_file").perform(context)
+    with open(path, "r") as f:
+        p = yaml.safe_load(f)["/**"]["ros__parameters"]
+    p["vehicle_length"] = p["front_overhang"] + p["wheel_base"] + p["rear_overhang"]
+    p["vehicle_width"] = p["wheel_tread"] + p["left_overhang"] + p["right_overhang"]
+    p["min_longitudinal_offset"] = -p["rear_overhang"]
+    p["max_longitudinal_offset"] = p["front_overhang"] + p["wheel_base"]
+    p["min_lateral_offset"] = -(p["wheel_tread"] / 2.0 + p["right_overhang"])
+    p["max_lateral_offset"] = p["wheel_tread"] / 2.0 + p["left_overhang"]
+    p["min_height_offset"] = 0.0
+    p["max_height_offset"] = p["vehicle_height"]
+    return p
+
+
+def get_vehicle_mirror_info(context):
+    path = LaunchConfiguration("vehicle_mirror_param_file").perform(context)
+    with open(path, "r") as f:
+        p = yaml.safe_load(f)
+    return p
+
+
+def create_additional_pipeline(vehicle_info, lidar_name):
+    cropbox_component = ComposableNode(
+        package="pointcloud_preprocessor",
+        plugin="pointcloud_preprocessor::CropBoxFilterComponent",
+        name=lidar_name + "_crop_box_filter",
```

Comment:
Both are okay but following style is a common.
```
f"{lidar_name}_crop_box_filter"
```

If you change coding style, please apply same change to other lines.

Reply:
fix in https://github.com/tier4/AutowareArchitectureProposal_launcher/pull/117/commits/c8731fb573aa47f554296bba0fae0074adc7c249